receiver/azuremonitorreceiver/scraper.go

```diff
@@ -241,6 +238,20 @@ func (s *azureScraper) getSubscriptions(ctx context.Context) {
 		return
 	}
 
+	if !s.cfg.DiscoverSubscriptions {
+		// if there's only subscription ids configured, we need to get additional info
+		for _, subID := range s.cfg.SubscriptionIDs {
+			resp, err := armSubscriptionClient.Get(ctx, subID, &armsubscriptions.ClientGetOptions{})
+			if err != nil {
+				s.settings.Logger.Error("failed to get Azure Subscription", zap.String("subscription_id", subID), zap.Error(err))
+				return
+			}
+			s.loadSubscription(resp.Subscription)
+		}
+		s.subscriptionsUpdated = time.Now()
```

Comment:
Related to previous comments, we should probably only mark subscriptionsUpdated if at least one subscription Get was successful (implying the remainder may have been deleted since configuring the receiver vs all failures is more likely a permissions or networking issue)

Reply:
From my previous comment, I think what I will do is that if people specify sub ids 
- if they don't want the name, I won't make the get
- if they want it, I will break if get doesn't work.

In that case, I you have a permission problem or anything, you can still disable the name attribute. 

WDYT?